### ISSUE TITLE
Tooltip Module Not Accessible

### ISSUE THREAD

**Issue by amberhinds** [WPDevelopers/essential-addons-for-elementor-lite#752]
Hello,

I was testing the accessibility of your plugin, specifically looking at the [tooltips](https://essential-addons.com/elementor/tooltip/).

These tooltips are missing some key elements to make them work for people with disabilities:

1. Tooltips need to be able to be triggered with a keyboard alone - currently you're adding tooltips to non-focusable elements (spans), but are not adding them into the focus order.
2. Your element that triggers the tooltip is missing aria-describedby connecting it to the tooltip so that screen reader users will be able to hear the tooltip.

Would you please fix these things so screen reader users and keyboard-only users can access the information too?

Here are some references that may be helpful:
- [Accessible tooltip codepen example](https://codepen.io/smhigley/pen/KjoerX)
- [ARIA tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)

**Comment by robinwpdeveloper** [comment]
Hi @amberhinds 
Thanks for reporting the issue.

We have fixed this on `3084` branch. 
And will be added in our upcoming release.

Feel free to share any feedback.
Cheers.

**Comment by amberhinds** [comment]
That's great! Thanks for the fix!

**Comment by rudlinkon** [comment]
This issue has already been fixed.
Thank you @amberhinds
